### DISCUSSION TITLE
Photosensitive Epilepsy Analysis Tool alternative

### DISCUSSION THREAD

**Discussion by evansque** [w3c/wcag#2562] [Show and tell]

**Comment by cstrobbe** [comment]
I assume this is about an alternative to the [Photosensitive Epilepsy Analysis Tool (PEAT)](https://trace.umd.edu/peat/) from Trace R & D. The only alternative that I'm aware of is [HardingFPA](https://www.hardingfpa.com/), which is probably not cheap.

**Comment by jacob-willden** [comment]
In addition to PEAT and the Harding Test, there's also [IRIS by Electronic Arts](https://github.com/electronicarts/IRIS) (BSD-3-Clause license), though the repository README specifies that IRIS, while meant to help reduce photosensitive epileptic risks, "is not intended to guarantee, certify or otherwise validate visual content's compliance with legal, regulatory or other requirements." Until [Dr. J. Bern Jordan completes D2](https://trace.umd.edu/open-source-photosensitive-epilepsy-analysis-tool/), IRIS appears to be the only free / open source software that checks videos for photosensitive epileptic risks.
